Source/SwiftLintBuiltInRules/Rules/Style/OpeningBraceRule.swift

```diff
@@ -206,14 +246,33 @@ private extension OpeningBraceRule {
             }
         }
 
-        private func refersToMultilineFunction(_ body: CodeBlockSyntax, functionIndicator: TokenSyntax) -> Bool {
+        private func isIgnoredMultilineFunc(_ body: CodeBlockSyntax, keywordToken: TokenSyntax) -> Bool {
+            guard configuration.allowMultilineFunc else {
+                return false
+            }
+
+            return isBodyPredecessorMultiline(body, keywordToken: keywordToken)
+        }
+
+        private func isIgnoredMultilineConditionStatement(_ body: some BracedSyntax, keywordToken: TokenSyntax) -> Bool {
```

Comment:
Since this function determines whether the `violationCorrection` function _should_ be called/processed, and in consideration of other suggestion, I believe something similar to the following would better reflect this function's intention/usage:
- `shouldIgnoreFromMultilineStatementConditions`